tests/clis/mlctl_init/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='sample_project',
+    version='0.1.0',
+    description="Python Boilerplate ML model contains all the boilerplate you need to create a working ML model project.",
+    packages=find_packages(),
+    install_requires=[
+            'mlsriracha>=0.0.2'
+        ],
+    python_requires='>=3.6',
+    entry_points={
+        'mlbaklava.train': [
+            'example_train = model.train:train'
+        ],
+        'mlbaklava.predict': [
+            'example_predict = model.predict:predict',
```

Comment:
For hosting, I think we are missing the load model on container start. Refer to the examples here:

https://github.com/intuit/baklava/tree/master/examples/7-predict-initializer